addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandlerFactory.java

```diff
@@ -31,8 +34,9 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.valloxmv")
 @NonNullByDefault
 public class ValloxMVHandlerFactory extends BaseThingHandlerFactory {
+    @Nullable
+    private WebSocketClient webSocketClient;
```

Comment:
Sry to be nit-picking but our guidelines tell us to place the annotation between the keyword `private` and `WebSocketClient`.

Maybe the better choice in this case is `@NonNullByDefault({})`.

Reply:
As I get for the second option an warning from eclipse which I was neither able to handle ("Nullness default is redundant with a default specified for the enclosing package org.openhab.binding.valloxmv.internal") nor I had an idea of the result, I implemented the first suggestion.